src/libos/src/vm/user_space_vm.rs

```diff
@@ -51,8 +51,25 @@ impl UserSpaceVMManager {
         self.range().size()
     }
 
-    pub fn inner(&self) -> &VMManager {
-        &self.inner
+    pub fn get_user_space_ranges(&self) -> Vec<VMRange> {
```

Comment:
If we always return two ranges, we can use array instead of vector as the return type.